### ISSUE TITLE
Add documentation for breathe-apidoc

### ISSUE THREAD

**Issue by samuel-emrys** [michaeljones/breathe#776]
There's currently no documentation for breathe-apidoc. This is a critical utility for automated documentation generation and should have it's options documented. I noticed that this is a common complaint, notably identified in #293 and #357. I thought I'd raise this for more discrete tracking of this issue - as it stands it wasn't obvious to me that this task was on the to-do list.

**Comment by vermeeren** [comment]
@samuel-emrys Yeah it's been on the TODO for very long by now, all it has now is `--help`'s output but no doc or mention anywhere else afaik.

Patches very welcome.
